Gemfile

```diff
@@ -34,7 +34,7 @@ gem 'nokogiri'
 gem "omniauth", ">= 2.0.0"
 gem 'omniauth-shibboleth', '~> 1.3'
 gem 'openurl'
-gem 'puma', '~> 3.7'
+gem 'puma'
```

Comment:
@bwatson78 do we foresee any scenarios where a upgrading to a major version could cause serious issues when we deploy? I am just wondering if it safer to lock to a major version at least and ensure any subsequent upgrades are backwards compatible until we explicitly upgrade to a new major version.

Reply:
Good point, let me check the Gemfile Lock real quick...